testing/chain.go

```diff
@@ -281,6 +288,9 @@ func (chain *TestChain) sendMsgs(msgs ...sdk.Msg) error {
 // number and updates the TestChain's headers. It returns the result and error if one
 // occurred.
 func (chain *TestChain) SendMsgs(msgs ...sdk.Msg) (*sdk.Result, error) {
+
+	chain.Coordinator.IncrementTime()
```

Comment:
this should be the only placed increment time needs to be done except for testing purposes